testdata/Caddyfile

```diff
@@ -23,6 +23,11 @@ http:// {
 		}
 		rewrite @indexFiles {http.matchers.file.relative}
 
+	@frankenphp path *.php {
+		root public/
+		index {path}/index.php 
+		resolve_root_symlink
+	}
```

Comment:
So the following lines can be removed, right?

Also, the indentation doesn't look good (you should be able to fix this automatically with [`caddy fmt`](https://caddyserver.com/docs/command-line#caddy-fmt).